Crashlytics/UnitTests/FIRCLSDataCollectionArbiterTest.m

```diff
@@ -49,12 +49,15 @@ - (void)tearDown {
   [super tearDown];
 }
 
-// If you do nothing, it should be YES. We should not be turning Fabric
-// customers off by default
 - (void)testNothingSet {
   self.fakeApp.isDefaultCollectionEnabled = YES;
   FIRCLSDataCollectionArbiter *arbiter = [self arbiterWithDictionary:@{}];
+#ifdef CRASHLYTICS_INTERNAL
+  XCTAssertFalse([arbiter isCrashlyticsCollectionEnabled]);
+#else
+  // It should be YES by default in Cocoapods.
```

Comment:
"...by default for 3P users"

(3P users can use Cocoapods, Carthage, etc.)

Reply:
Ah - yes! Thanks!